assignment-6&7/src/App.js

```diff
@@ -1,4 +1,5 @@
 import { useEffect, useState } from 'react';
+import { Outlet } from 'react-router-dom';
```

Comment:
can give an empty line between libraray imports and local file imports for better readability.